### ISSUE TITLE
More Modifiers 

### ISSUE THREAD

**Issue by Caldeiradas** [SlimeKnights/TinkersConstruct#3701]
Tinkers is, without doubt, one of my favorite mods or even what I like most: either by the weapons or the modifiers and the powers you end up with. The most beautiful parts are undoubtedly the long range weapons and to these I think it may make sense to add more modifiers.

### Punch Through
There are elements that can make projectiles travel instantly but do not trespass mobs, which is bad because if that happened it was completely fantastic. When we talk about this seems to be too powerful but it is supposed to be an end game weapon and even so not all monsters can have this: bosses should not trespass.
In addition to this they can have the damage can be reduced by each mob that passes. If you still think this is too OP then play with Scaling Health

### Multi-shot
And what about adding some kind of glass or even a mirror that copies the projectile and that throw together with the main projectile another set of x (+1 for each added spit). Each projectile gives 50% of main damage but copies all statuses and has 80% accuracy versus main.

It's supposed to be an early game thing but not too easy to get.

Craft Ideas: `Bone meal` + `glass pane` + `GlowStone poder`

### Explosive projectiles
Play ONCE with this and the mod that adds the metal crashed or was discontinued and I never managed to do it again but is a fantastic idea!
It is the only modifier that spends durability even if it is unbreakable. It is added by adding a `Flint and Steel` and `gunpowder`. Such as `Lapis Lazulis` and `RedStone`, the amount of `gunpowder` will indicate the level and the `Flint and Steel` may: either lose durability by use or be even consumed. It has a 40% probability of exploding and a further 20% if the mob is on fire. adding more of this just increases the damage and / or range of the explosion.

### Damage Range and Multiplier
For long range weapons we have the ability to reduce the firing time but we have no way to increase the range or multiplier damage! It's sad but you may have an idea for it.

To achieve it should be something that would make sense even for the shrikes and possibly other weapons if they develop for MC 1.13. However we may have a change in the effect of how the weapon is used.
Weapons that use throwing force do not have a damage multiplier so these two can be the same: `Power Potion`.

For the range of bows and crossbows can be increased by adding `vanilla arches` + `slime balls` as if the reach of these arches were added to the weapon in question.

To be clearer:
* `Power Potion`
  * shurikens → increase range
  * throwing weapons → increases reach
  * arcs → increases damage multiplier
  * Beasts → increases damage multiplier
* `Bow + slime`
  * shurikens → not applicable
  * Shooting weapons → not applicable
  * Arcs → increase range
  * crossbows → increase range

**Comment by SweetSugarSugar** [comment]
Suggestions are generally not taken.

Crossbows and bows aren't meant to be laser guns.  There is an addon for that though if that's what you're interested in.

Splitting is already in the mod as bone's arrow shaft trait.

Exploding weapon projectiles don't really fit the balance of Tinkers' either.  There is however EFLNs which can be used for mining and such.

Damage, range, and reload for bows/crossbows is dependent on the material choice and the bolts being fired.  Shurikens are also meant to be short-range weapons.

Overall, most of the suggestions made, while unique, are mostly overpowered.  TiC 2 is all about balancing material choices and traits rather than modifiers.

**Comment by Caldeiradas** [comment]
Well then ... Thanks any way.  
Some times I can not stop my self from imagining and as so I had the idea of leaving this here.

By the way: is there any way I can change the configurations so I can add limits for some modifiers?

**Comment by SweetSugarSugar** [comment]
No, but there might be a config option for Tinkers' Tool Leveling if you want to limit the amount of overall modifiers.
